server/src/main/java/org/elasticsearch/index/IndexVersions.java

```diff
@@ -143,6 +143,7 @@ private static Version parseUnchecked(String version) {
     public static final IndexVersion INFERENCE_METADATA_FIELDS = def(9_005_00_0, Version.LUCENE_10_0_0);
     public static final IndexVersion LOGSB_OPTIONAL_SORTING_ON_HOST_NAME = def(9_006_00_0, Version.LUCENE_10_0_0);
     public static final IndexVersion SOURCE_MAPPER_MODE_ATTRIBUTE_NOOP = def(9_007_00_0, Version.LUCENE_10_0_0);
+    public static final IndexVersion TIMESTAMP_DOC_VALUES_SPARSE_INDEX = def(9_008_00_0, Version.LUCENE_10_0_0);
```

Comment:
After merging https://github.com/elastic/elasticsearch/pull/120741 I will need to increase this to `9_010_00_0` or whatever depending on other PRs merged in the meanwhile.